Cargo.toml

```diff
@@ -23,7 +23,7 @@ members = [
     "infrastructure/shutdown",
     "infrastructure/storage",
     "infrastructure/test_utils",
-    "applications/tari_testnet_miner",
+    #"applications/tari_testnet_miner",
```

Comment:
Because this will change quite a bit, I have removed this for now from compiling